task_spec/spec/acceptance/init_spec.rb

```diff
@@ -45,6 +45,7 @@ def latest_platform_list
     %r{
       el-9-aarch64|
       ubuntu-22\.04-aarch64|
```

Comment:
here you match for aarch64, should that be cleaned up?

Reply:
I am not sure what you are asking here but cleanup for released platforms can be done with separate jira.
https://perforce.atlassian.net/browse/PA-6627

Reply:
The reason we're not including aarch for Ubuntu 24 is because both x86_64 and aarch are newly supported platforms and haven't had public releases yet